releaseNotes/PythonFeatureManagement.md

```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b2 - October 14, 2024
+
+### Features
+
+* Added VariantAssignmentPercentage, DefaultWhenEnabled to telemetry data when using `publish_telemetry`.
```

Comment:
Not sure how it's related to `publish_telemetry`. Shouldn't these be included only when telemetry is enabled?

Reply:
The `publish_telemetry` is what we ship for Azure Monitor, right? Does this sound reasonable?

Added support for including `VariantAssignmentPercentage` and `DefaultWhenEnabled` properties to telemetry when telemetry is enabled and published to Azure Monitor.